bottle_app.py

```diff
@@ -468,7 +468,7 @@ def app():
                                      formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
     parser.add_argument('--dbcredentials',help='Specify .ini file with [dbreader] and [dbwriter] sections')
-    parser.add_argument('--port',type=int)
+    parser.add_argument('--port',type=int,default=8008)
```

Comment:
The current default port for bottle is 8080, and this is 8008. Was that intentional or a typo?

Reply:
Typo. I'll fix.